### ISSUE TITLE
How to create a custom function in a functionRegistry

### ISSUE THREAD

**Issue by dmziryanov** [ONLYOFFICE/sdkjs-plugins#146]
How to create a custom simple function, for a example foo() to insert in a cell '=foo()'?
Is it possible without modification the source code?

**Comment by ShockwaveNN** [comment]
@askonev Please take a look

**Comment by askonev** [comment]
Hello, @dmziryanov 
Perhaps the use of our macros will help you to solve your problem.
Example. Let's find with the help of macros the minimum value from the range of cells from B1 to B3.

![Screen record from 2021-03-05 11 20 43](https://user-images.githubusercontent.com/60688343/110087599-e8df7800-7da4-11eb-9a3f-ad3720acf3e0.gif)

>Script
```js
(function()
{
    oWorksheet = Api.GetActiveSheet();
    oWorksheet.GetRange("A1").SetValue("=min(B1:B3)");
})();
```

**Comment by dmziryanov** [comment]
I understood that's possible only with modification the source code.

Plugins and macroses are great to help, but the best way to solve our problem is to push a custom function (e.g. foo(A2, B2)) through the table and fill certain cells. 

This function is impossible to create like combination of the existing functions because it should take data from REST API with token authorization and should be defined in such way that cell analizer could evaluate it like other functions.

As far as I could understood this is imposible without modification ONLYOFFICE/core

**Comment by askonev** [comment]
@dmziryanov Yes, you're right about the fact that adding such functionality will require a modification ONLYOFFICE/core. However, how do you look at the analog of the [LAMBDA functions](https://support.microsoft.com/en-us/office/lambda-function-bd212d27-1cd1-4321-a34a-ccbf254b8b67). At the moment we do not have this functionality, but we will think about its implementation.
